openapi/src/json-schemas/api/resources/metadata-resource.json

```diff
@@ -0,0 +1,48 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Metadata resource",
+    "allOf": [
+        {"$ref": "../json-api-base-resource.json"},
+        {
+            "properties": {
+                "type": {
+                    "const": "metadata"
+                },
+                "attributes": {
+                    "allOf": [
+                        {
+                            "required": [
+                                "questionnaire-document-version",
+                                "questionnaire-id",
+                                "created",
+                                "modified",
+                                "submission-status",
+                                "expires"
+                            ],
+                            "properties": {
+                                "questionnaire-document-version": {
+                                    "enum": ["4.1.0"]
```

Comment:
should this be abstracted out in its own schema to avoid this value deviating from the truth? 🤔 using the $ref syntax, and referencing a questionnaire-document-version schema with a static enum as a property.

The [latest ](https://github.com/CriminalInjuriesCompensationAuthority/q-templates-application/blob/master/src/template.js#L714)is `4.2.0`

Reply:
I've updated this to 4.2.0.

I don't think it would make sense to abstract this out. We're not re-using the schema anywhere else, plus we'd still need to remember to keep it in sync. I don't think we'd gain much and it'd be inconsistent with all the other `enum` schema we've used.